CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.1.0 - 21 Apr 2020
+### Added
```

Comment:
'Add Docker Compose...' ?

Reply:
We've had the ability to add compose files for a very long time; that it is now split into another command isn't particularly interesting to users.